smart_pandas/smart_pandas.py

```diff
@@ -29,3 +38,25 @@ def model_features(self):
     @property
     def target(self):
         return self._obj[self.config.target]
+
+    @property
+    def schema(self):
+        return self.config.schema
+
+    def validate(self, **kwargs):
+        """
+        Validate the DataFrame using the pandera schema defined in the config.
+
+        Parameters
+        ----------
+        kwargs
+            Additional keyword arguments to pass to the pandera schema validate method.
+
+        Returns
+        -------
+        validated_data: pd.DataFrame
+            The validated DataFrame.
+        """
+        validated_data = self.config.schema.validate(self._obj, **kwargs)
+        validated_data.config.init(config=self.config)
+        return validated_data
```

Comment:
Thoughts on adding a pandas like `inplace` argument here? The call can be:

```python
data.config.validate(inplace=True)
```

And data now contains the new coerced types

Reply:
Added with 1fa4a09

Reply:
🚀